starter.js

```diff
@@ -0,0 +1,178 @@
+// PART 0: Just DO it!!
+function justDoIt(yourName) {
+    if (typeof(yourName) == 'string'){
+    return `${yourName} , JUST DO IT!`; 
+    }
+}   console.log(justDoIt('muneera'));
+
+
+// PART 1: Big or Small String?
+function bigOrSmallString(word){
+    if (typeof(word) == 'string' && word.length > 10 ){
+        return `${word} This word is loooooong` ;
+    }else if (typeof(word) == 'string' && word.length < 10){
+        return `${word} This word is short 😬` ;
+    } else {
+        return `Enter a valid string. ` ;
+    }
+} 
+console.log(bigOrSmallString('muneera'));
+console.log(bigOrSmallString('muneerrrraaaaa'));
+console.log(bigOrSmallString(99));  
+
+
+// PART 2: Odd or Even String Length
+function oddOrEvenString(word2){
```

Comment:
you can use just `word` as a parameter it won't collide with `bigOrSmallString` function